src/Components/SmartComponents/SystemDetailsPage/SystemDetailsPage.js

```diff
@@ -102,13 +103,27 @@ class InventoryDetail extends React.Component {
     render() {
         const { InventoryHeader, InventoryBody, InvWrapper } = this.state;
         const { systemDetails = {}, errors } = this.props;
-        const { opt_out: isOptOut = false, entity } = systemDetails;
-
+        const { opt_out: isOptOut = false, entity, loaded } = systemDetails;
         const Wrapper = InvWrapper || React.Fragment;
         return (
             <Page>
                 <Wrapper>
-                    <VulnerabilityHeader title=''>
+                    <Header
+                        title=''
+                        breadcrumbs={[
+                            {
+                                title: PATHS.systemsPage.title,
+                                to: PATHS.systemsPage.to,
+                                loaded: true
+                            },
+                            {
+                                // eslint-disable-next-line camelcase
+                                title: entity?.display_name || this.props.intl.formatMessage(messages.invalidSystem),
```

Comment:
I hate how lint considers this variable and complains about it not being camelcase and then we have to suppress it. Do you think we can somehow fix this to the future (maybe it's a bug, maybe there's an option for that)? I think we'll be using optional chaining one quite a lot and this will be annoying.

Reply:
same feeling here! I will take a look if we could update the eslint rules for that